src/SDSS.jl

```diff
@@ -1,21 +1,26 @@
 module SDSS
 
+VERSION < v"0.4.0-dev" && using Docile
 using CelesteTypes
+using FITSIO
+using WCSLIB
+using DataFrames
+using Grid
+using GaussianMixtures
 
 # FITSIO v0.6.0 removed some of the lower-level functions used here.
 # In particular, I think wcslib may still need this.
```

Comment:
wcslib seems to work without lower-level FISIO, at least on my system. Are you using the latest version of wcslib?